sycl/test/extensions/properties/properties_kernel.cpp

```diff
@@ -8,12 +7,12 @@ using namespace sycl::ext::oneapi::experimental;
 
 using device_has_all =
     decltype(device_has<
+             // expected-warning@+1{{'host' is deprecated: deprecated in SYCL 2020, 'host' device concept removed}}
```

Comment:
We have dedicate tests for deprecation warnings in `warnings/` subdirectory. I would prefer to move this check there. We could use `-Wno-deprecated` for this test to keep `expected-no-diagnostics` in place.